NoteFind/Alerts/ErrorAlerts.h

```diff
@@ -0,0 +1,33 @@
+//
+//  ErrorAlerts.h
+//  NoteFind
+//
+//  Created by Khloe Wright on 7/22/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "LoginViewController.h"
+#import "FeedViewController.h"
+#import "RegisterViewController.h"
+#import "NewNoteViewController.h"
+#import <UIKit/UIKit.h>
+
+@class LoginViewController;
+@class FeedViewController;
+@class RegisterViewController;
+@class NewNoteViewController;
+
```

Comment:
Handful of feedback about this header.

- When you use `@class Foo`, you don't need to import `Foo.h`. The `@class` basically hints to the compiler to trust that the type actually exists and you'll likely need this import in the implementation file.
- Style nit: Try to organize your imports so that apple frameworks, third party framework headers are above headers for your app, it makes it easier to scan.